smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -1,192 +1,50 @@
-task Star {
-  File input_fastq_read1
-  File input_fastq_read2
-  File gtf
-  File star_genome
-
-  command {
-    tar -xvf ${star_genome}
-    STAR  --readFilesIn ${input_fastq_read1} ${input_fastq_read2} \
-      --genomeDir ./star \
-      --quantMode TranscriptomeSAM \
-      --outSAMstrandField intronMotif \
-      --genomeLoad NoSharedMemory \
-      --sjdbGTFfile ${gtf} \
-      --readFilesCommand "zcat" \
-      --twopassMode Basic \
-      --outSAMtype BAM SortedByCoordinate  \
-      --outSAMunmapped Within \
-      --limitBAMsortRAM 30000000000 
-  }
-  output {
-    File junction_table = "SJ.out.tab"
-    File output_bam = "Aligned.sortedByCoord.out.bam"
-    File output_bam_trans="Aligned.toTranscriptome.out.bam"
-  }
-  runtime {
-    docker:"humancellatlas/star:2.5.3a"
-    memory: "40 GB"
-    disks :"local-disk 100 HDD"
-  }
-}
-
-task FeatureCountsUniqueMapping {
-  File aligned_bam
-  File gtf
-  String fc_out
+import "hisat2_QC_pipeline.wdl" as run_hisat2
```

Comment:
Please add more comments about why you are running hisat2_QC and also hisat2_rsem -- I doubt this is transparent to users.